openedx/features/custom_fields/multiselect_with_other/forms/fields.py

```diff
@@ -37,8 +38,10 @@ def valid_value(self, value):
         return len(value) <= self.other_max_length
```

Comment:
`openedx/features/custom_fields/multiselect_with_other/forms/fields.py:32: [E1101(no-member), MultiSelectWithOtherFormField.__init__] Instance of '__proxy__' has no 'format' member`

One of the pending linting error, seems a False positive. we can suppress the warning by adding at the end of line 32

(add two spaces before this) `# pylint: disable=E1101` 


What do you think @saad-waqar

Reply:
Agreed